.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -2,7 +2,8 @@
 name: Bug report
 about: Create a report to help us improve
 title: ''
-labels: 'bug'
```

Comment:
Should we keep both the label and the type for now until we transition all old issues (including ones not tracked in projects)?